pkgs/by-name/st/stl-to-obj/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  cmake,
+  fetchFromGitHub,
+  fetchpatch,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "stl-to-obj";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "Neizvestnyj";
+    repo = "stl-to-obj";
+    rev = finalAttrs.version;
+    hash = "sha256-+R7rNpMxKFC7sLYQXZX3Ikb5MqNd57r1M8gma73kCcg=";
+  };
+
+  patches = [
```

Comment:
Since these patches point to commits not yet included in the upstream repo, I think it would be better to include them in nixpkgs (they may disappear if you force-push to the PR branches)

Reply:
I did replace those with simple bash things, if this is OK for you